pages/beta/admin/harvesters/[id]/index.vue

```diff
@@ -162,7 +162,7 @@
 </template>
 
 <script setup lang="ts">
-import { Pagination } from '@datagouv/components'
+import { Pagination } from '@datagouv/components-next'
```

Comment:
SimpleBanner is missing

Reply:
I don't see `SimpleBanner` use in this file?